### PR TITLE
Initial build

### PR DESCRIPTION
**Pull request by codytipton**
v1.0

### CODE REVIEW

File: analyticConfigurations/metric_meta-consul.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	  "type": "COUNTER"
+	  }
+      },
+      {
+	"match": "consul.serf.*",
+	"properties": "MAX",
+	  "statistic": "MAX",
+	  "type": "COUNTER"
+	  }
+      }
```
Comment (TheConnMan) [inline analyticConfigurations/metric_meta-consul.json:39]: Invalid JSON.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
+{
+  "id" : "netuitive.packages.linux.consul",
+  "version" : "0.1",
```
Comment (TheConnMan) [inline package.json]: Could you set this to `0.1.0`?

File: analyticConfigurations/metric_meta-consul.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (TheConnMan) [inline analyticConfigurations/metric_meta-consul.json:1]: Can you use the Metricly CLI to format this package?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,11 @@
 # netuitive-packages-linux-consul
+
 A set of Netuitive analytics configurations, polices, dashboards, and reports that are used to monitor performance of Consul resources.
+
+## Release History
+
+### Version next
+
+### Version 1.0.0
```
Comment (TheConnMan) [inline README.md]: Can you put this under `next` then we can cut a `1.0.0` directly on dev?

**Review by TheConnMan — CHANGES_REQUESTED** [review]

File: analyticConfigurations/metric_meta-consul.json (hunk 3 above)
Comment (codytipton) [reply analyticConfigurations/metric_meta-consul.json:1]: That is very handy. Will do from now on.

File: analyticConfigurations/computed_metric-consul.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "consul.raft.commitTime.max",
+        "properties": {
+          "expression": "metric['consul.raft.commitTime'].hmax",
```
Comment (jemassey) [inline analyticConfigurations/computed_metric-consul.json]: This is the historical maximum, the highest value seen in the history of this metric... not sure if that is what you actually want here?

Comment (jemassey) [reply analyticConfigurations/computed_metric-consul.json]: should just be `.max` I believe

**Review by Grumblesmith — CHANGES_REQUESTED** [review]
`metricly package format ` will need to be run against the following files before the build will pass.
policies/linux.consul.-.catalog.nodes_critical.json
policies/linux.consul.-.catalog.nodes_passing.json
analyticConfigurations/ace-consul.json
analyticConfigurations/computed_metric-consul.json
analyticConfigurations/metric_meta-consul.json

File: policies/linux.consul.-.catalog.nodes_critical.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 1,
+        "metric": "consul.catalog.nodes_critical",
+        "operator": "<"
+      }
+    ],
+    "deleted": false,
+    "description": "consul.catalog.nodes_critical is greater than 0.",
```
Comment (rclagett) [inline policies/linux.consul.-.catalog.nodes_critical.json]: Typically, we explain why this alert should be paid attention. Can you provide more details in the description?

File: policies/linux.consul.-.catalog.nodes_passing.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 1,
+        "metric": "consul.catalog.nodes_passing",
+        "operator": "<"
+      }
+    ],
+    "deleted": false,
+    "description": "Number of nodes with service status `passing` same as those registered",
```
Comment (rclagett) [inline policies/linux.consul.-.catalog.nodes_passing.json]: Can you provide more detail in this description as well?

**Review by rclagett — CHANGES_REQUESTED** [review]

**Review by TheConnMan — APPROVED** [review]

**Pull request merged**